test/client_test.go

```diff
@@ -93,7 +95,10 @@ func TestDefaultClientRetries(t *testing.T) {
 	c := client.New(testServer.URL)
 	_, err := c.GetBooks(context.Background(), &models.GetBooksInput{})
 	assert.NoError(t, err)
-	assert.Equal(t, 2, controller.getCount)
+	backoff := controller.getTimes[1].Sub(controller.getTimes[0])
+	if backoff < 90*time.Millisecond || backoff > 110*time.Millisecond {
```

Comment:
i recently learned about https://github.com/stretchr/testify/blob/master/assert/assertions.go#L722-L727 which makes this assert a little nicer